src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -8,151 +8,134 @@ if [ ! -d "$PIP_CACHE_DIR" ];then
+		fi
+	elif [ -e "setup.py" ]
 	then
-		exit $pythonBuildExitCode
+		echo "Running python setup.py install..."
+		python setup.py install --user| ts $TS_FMT
+		cd *.egg-info
+		python -m pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r requires.txt | ts $TS_FMT
+
+		pythonBuildExitCode=${PIPESTATUS[0]}
+		if [[ $pythonBuildExitCode != 0 ]]
+		then
+			exit $pythonBuildExitCode
+		fi
+	else
+		echo $REQS_NOT_FOUND_MSG
 	fi
-else
-	echo $REQS_NOT_FOUND_MSG
-fi
-
-# For virtual environment, we use the actual 'python' alias that as setup by the venv,
-python_bin=python
 
+	# For virtual environment, we use the actual 'python' alias that as setup by the venv,
+	python_bin=python
 {{ else }}
-
-if [ -e "requirements.txt" ]
-then
-	echo
-	echo Running pip install...
-	START_TIME=$SECONDS
-	$pip install  --cache-dir $PIP_CACHE_DIR --prefer-binary -r requirements.txt --target="{{ PackagesDirectory }}" --upgrade | ts $TS_FMT
-	pipInstallExitCode=${PIPESTATUS[0]}
-	ELAPSED_TIME=$(($SECONDS - $START_TIME))
-	echo "Done in $ELAPSED_TIME sec(s)."
-
-	if [[ $pipInstallExitCode != 0 ]]
-	then
-		exit $pipInstallExitCode
-	fi
-elif [ -e "setup.py" ]
-then
-	echo
-	START_TIME=$SECONDS
-	pip install --upgrade pip
-	ELAPSED_TIME=$(($SECONDS - $START_TIME))
-	echo "Done in $ELAPSED_TIME sec(s)."
-
-	echo "Running python setup.py install..."
-	$python setup.py install --user| ts $TS_FMT
-	cd *.egg-info
-	pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r requires.txt | ts $TS_FMT
-	pythonBuildExitCode=${PIPESTATUS[0]}
-	if [[ $pythonBuildExitCode != 0 ]]
-	then
-		exit $pythonBuildExitCode
-	fi
-else
-	echo $REQS_NOT_FOUND_MSG
-fi
-
-# We need to use the python binary selected by benv
-python_bin=$python
-
-# Detect the location of the site-packages to add the .pth file
-# For the local site package, only major and minor versions are provided, so we fetch it again
-SITE_PACKAGE_PYTHON_VERSION=$($python -c "import sys; print(str(sys.version_info.major) + '.' + str(sys.version_info.minor))")
-SITE_PACKAGES_PATH=$PIP_CACHE_DIR"/lib/python"$SITE_PACKAGE_PYTHON_VERSION"/site-packages"
-mkdir -p $SITE_PACKAGES_PATH
-# To make sure the packages are available later, e.g. for collect static or post-build hooks, we add a .pth pointing to them
-APP_PACKAGES_PATH=$(pwd)"/{{ PackagesDirectory }}"
-echo $APP_PACKAGES_PATH > $SITE_PACKAGES_PATH"/oryx.pth"
-
-{{ end }}
-
-{{ if EnableCollectStatic }}
-if [ -e "$SOURCE_DIR/manage.py" ]
-then
-	if grep -iq "Django" "$SOURCE_DIR/requirements.txt"
+	if [ -e "requirements.txt" ]
 	then
 		echo
-		echo Content in source directory is a Django app
-		echo Running 'collectstatic' ...
+		echo Running pip install...
 		START_TIME=$SECONDS
-		$python_bin manage.py collectstatic --noinput || EXIT_CODE=$? && true ; 
-		echo "'collectstatic' exited with exit code $EXIT_CODE."
+		$python -m pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r requirements.txt --target="{{ PackagesDirectory }}" --upgrade | ts $TS_FMT
```

Comment:
Using `$python` instead of `python` here since it is _not_ run from within a virtual environment